src/less/main.less

```diff
@@ -0,0 +1,190 @@
+@some-grey: #727272;
+@some-other-grey: #dfdfdf;
+
+.transition-all {
+  transition-property: all;
+  transition-duration: 0.2s;
+  transition-timing-function: ease;
+}
+
+.rise-enabled {
+  background-color: #fff;
+  .transition-all;
+
+  /* Reveal-specific */
+  .reveal {
+    &.fade {
+      .transition-all;
+      opacity: 1;
+    }
+
+    /* element-overrides */
+    pre {
+      .transition-all;
+      font-size: 90%;
+      box-shadow: 0px 0px 0px rgba(0, 0, 0, 0);
+      overflow: hidden;
+    }
+
+    section img {
+      .transition-all;
+      border: 0px solid black;
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0);
+      transition: none;
+    }
+
+    blockquote {
+      p:first-child, p:last-child { display: inherit; }
+    }
+
+    .slides {
+      .transition-all;
+      text-align: left;
+      font-size: 160%;
+    }
+
+    .slide-number {
+      .transition-all;
+      right: 3.1em;
+      bottom: 3.9em;
+      font-size: 100%;
+    }
+
+    .controls {
+      .navigate-left, .navigate-left.enabled{
+        border-right-color: @some-grey;
+        &.enabled:hover{ border-right-color: @some-other-grey; }
+      }
+      .navigate-right, .navigate-right.enabled{
+        border-left-color: @some-grey;
+        &.enabled:hover{ border-left-color: @some-other-grey; }
+      }
+      .navigate-up, .navigate-up.enabled{
+        border-bottom-color: @some-grey;
+        &.enabled:hover{ border-bottom-color: @some-other-grey; }
+      }
+      .navigate-down, .navigate-down.enabled{
+        border-top-color: @some-grey;
+        &.enabled:hover{ border-top-color: @some-other-grey; }
+      }
+    }
+
+    .progress {
+      margin-bottom: 0;
+      span { background: @some-grey; }
+    }
+
+    .reveal-skip, .reveal-notes { display: none; }
+
+  } // .reveal
+
+  /* Notebook-specific */
+  #notebook { overflow: hidden; }
+
+  #notebook.reveal {
+    padding-top: 0px;
+  }
+
+  #notebook-container {
+    .transition-all;
+    background-color: transparent;
+    box-shadow: none;
+    width: 100%;
+  }
+
+  .ctb_global_show {
+    .ctb_hideshow.ctb_show { display: none; }
```

Comment:
this seems to be not working, because if I enter the slideshow mode with the ctb active, I still see it...